core/src/main/java/dev/jsinco/brewery/util/MessageUtil.java

```diff
@@ -27,6 +32,22 @@ public class MessageUtil {
 
     private static final char SKULL = '\u2620';
 
+    public static Component mm(String msg) {
+        return MiniMessage.miniMessage().deserialize(msg);
+    }
+    
+    public static Component mm(String msg, TagResolver... resolvers) {
+        return MiniMessage.miniMessage().deserialize(msg, resolvers);
+    }
+
+    public static void msg(Audience audience, String msg) {
+        audience.sendMessage(mm(msg));
+    }
+
+    public static void msg(Audience audience, String msg, TagResolver... resolvers) {
+        audience.sendMessage(mm(msg, resolvers));
+    }
```

Comment:
mm is unclear what it means, msg is a known abbreviation, so that one is probably alright, but I prefer none of those.
```suggestion
    public static Component miniMessage(String msg) {
        return MiniMessage.miniMessage().deserialize(msg);
    }
    
    public static Component miniMessage(String msg, TagResolver... resolvers) {
        return MiniMessage.miniMessage().deserialize(msg, resolvers);
    }

    public static void message(Audience audience, String msg) {
        audience.sendMessage(mm(msg));
    }

    public static void message(Audience audience, String msg, TagResolver... resolvers) {
        audience.sendMessage(mm(msg, resolvers));
    }
```